include/nnvm/compiler/packed_func_ext.h

```diff
@@ -52,6 +54,12 @@ template<>
 struct extension_class_info<nnvm::compiler::AttrDict> {
   static const int code = 18;
 };
+
+template<>
+struct extension_class_info<nnvm::compiler::SymbolArray> {
+  static const int code = 19;
```

Comment:
cannot use 19, because it is taken by mxnet tvm bridge https://github.com/apache/incubator-mxnet/blob/master/include/mxnet/tensor_blob.h#L49

Reply:
It seems to be quite dangerous to expose a vector of pointers here.